src/Models/Properties/ElementProperties/LookupProperty.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace form_builder.Models.Properties.ElementProperties
+{
+    public partial class BaseProperty
+    {
+        public List<Lookup> Lookup { get; set; }
+    }
```

Comment:
my only question is if this should be renamed from Lookup as it might be confusing have Lookup twice within Properties and in parent. Maybe LookupSource?